langs/es.ini

```diff
@@ -1,11 +1,19 @@
-name="Español"
+name="Alemán"
 dir="ltr"
-missing="Lamentableme no hay traduccion disponible"
+missing="Lamentablemente no hay ninguna traducción disponible"
 readmore="Leer más"
+languages="Idiomas"
+section="Sección"
+shortcuts="Abreviaturas"
+legal="legal"
 email="Correo electrónico"
 fax="Fax"
 telephone="Teléfono"
 bus="Autobús"
 subway="Metro"
+skip="Ir al contenido"
+newsletter="Suscribirse al boletín de noticias"
```

Comment:
This doesn't look right.

Reply:
Why? It corresponds to the German source "Zum Inhalt springen" and "Newsletter abonnieren"

Reply:
Ah, you are right, sorry. I thought had ought to be a translation of "newsletter". My mistake.